src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -15,48 +22,109 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import FormControl from '@mui/material/FormControl';
 import IconButton from '@mui/material/IconButton/IconButton';
+import IndeterminateCheckBoxIcon from '@mui/icons-material/IndeterminateCheckBox';
 import Input from '@mui/material/Input';
 import InputAdornment from '@mui/material/InputAdornment';
 import InputLabel from '@mui/material/InputLabel';
 import MenuIcon from '@mui/icons-material/Menu';
 import MailIcon from '@mui/icons-material/Mail';
 import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MarkEmailReadIcon from '@mui/icons-material/MarkEmailRead';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import PhotoCameraIcon from '@mui/icons-material/PhotoCamera';
+import SendIcon from '@mui/icons-material/Send';
+import SettingsIcon from '@mui/icons-material/Settings';
+import StarsIcon from '@mui/icons-material/Stars';
+import Switch from '@mui/material/Switch';
+import Typography from '@mui/material/Typography';
 import useStyles from './SearchChannel.styles';
 import './styles.css';
 
 const ChannelList = () => {
   const [drawer, setDrawer] = useState(false);
+  const [isShown, setIsShown] = useState(false);
+
   const styles = useStyles();
 
   return (
     <>
       <Drawer anchor="left" open={drawer} onClose={() => setDrawer(false)}>
-        <Box sx={{ width: 250 }} role="presentation">
+        <Box sx={{ width: 250 }} role="presentation" className={styles.upperGroup}>
           <Box className={styles.account}>
-            <Avatar alt="Channel Avatar" src={ReneAvatar} className={styles.accountAvatar} />
+            <Button
+              component="label"
+              onMouseEnter={() => setIsShown(true)}
+              onMouseLeave={() => setIsShown(false)}
+            >
+              <Avatar alt="Channel Avatar" src={ReneAvatar} className={styles.accountAvatar} />
+              <input hidden accept="image/*" multiple type="file" />
+            </Button>
           </Box>
-          <List>
-            {['Inbox', 'Starred', 'Send email', 'Drafts'].map((text, index) => (
-              <ListItem key={text} disablePadding>
-                <ListItemButton>
-                  <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon>
-                  <ListItemText primary={text} />
-                </ListItemButton>
-              </ListItem>
-            ))}
-          </List>
           <Divider />
           <List>
-            {['All mail', 'Trash', 'Spam'].map((text, index) => (
-              <ListItem key={text} disablePadding>
-                <ListItemButton>
-                  <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon>
-                  <ListItemText primary={text} />
-                </ListItemButton>
-              </ListItem>
-            ))}
+            <ListItem disablePadding>
```

Comment:
This is just a suggestion. To me, this is a good example to create a component so we can reuse most of the boilerplate code, since we are repeating ourselves too much here ;)